tests/missing_newline/metadata.json

```
@@ -0,0 +1,16 @@
+{
+  "name": "puppetlabs-postgresql",
+  "version": "3.4.1",
+  "author": "Inkling/Puppet Labs",
+  "summary": "A description with an invalid \( escape sequence",
```

Perhaps leave this out just so it only tests the specific failure mode?
```suggestion
  "summary": "A metadata.json file without a newline at the end",
```

---

ah ups :D